src/elm/Page/Account.elm

```diff
@@ -414,6 +434,66 @@ update msg env model =
                             |> addValidationError Consent "Fikk ikke til å lagre samtykke."
                             |> PageUpdater.init
 
+        GetRecurringPayments ->
+            PageUpdater.fromPair ( model, getRecurringPayments env )
+
+        ReceiveRecurringPayments result ->
+            case result of
+                Ok recurringPayments ->
+                    PageUpdater.init
+                        { model | recurringPayments = Loaded recurringPayments }
+
+                Err _ ->
+                    -- TODO: There is an issue with this call being made two times when a user
+                    -- refreshes and thus we can't handle this properly currently. A refactor of
+                    -- the way we handle user data and tokens will fix this.
+                    let
+                        errorMessage =
+                            "Fikk ikke hentet lagrede betalingskort."
+                    in
+                        PageUpdater.init { model | recurringPayments = Failed errorMessage }
+
+        EndRecurringPayment recurringPayment ->
```

Comment:
Burde ikke dette være `DeleteRecurringPayment`? Kunne vært enig med "end" hvis det faktisk var et abonnement som skulle avsluttes, men recurring payments brukes (også) for å bare lagre bankkort.

Reply:
Generelt er begrepsbruken fra Entur benyttet over hele linja for dette med recurring-payment, og det gjelder også bruken av begrepet "end". Fra deres doc:
> Ends a recurring payment, the recurring payment is no longer usable after this call.

Grunnen til at de bruker "end" i stedet for "delete" er nok fordi i praksis blir ikke noe slettet, men endrer status fra `ACTIVE` til `CANCELLED`.

Jeg skal ikke stå på barrikadene for å benytte "end" i stedet for "delete", men synes det er mest konsekvent når det gjelder det å holde seg til Entur sin terminologi for dette med lagrede kort.

Reply:
Det hjøres jo til rart ut i så fall, for da lagrer de sensitive data utover det kunden forventer. Jeg kan skjønne ordbruken med end hvis det faktisk var et abonnement. Men ok, hvis de bruker end kan vi også gjøre det.